src/test/run-make/thumb-none-qemu/example/Cargo.toml

```diff
@@ -2,10 +2,10 @@
 name = "example"
 version = "0.1.0"
 authors = ["Hideki Sekine <sekineh@me.com>"]
-# edition = "2018"
+edition = "2018"
 
 [dependencies]
-cortex-m = "0.5.4"
-cortex-m-rt = "=0.5.4"
+cortex-m = "0.6.0"
+cortex-m-rt = "=0.6.10"
 panic-halt = "0.2.0"
-cortex-m-semihosting = "0.3.1"
+cortex-m-semihosting = "0.3.3"
```

Comment:
Is there a reason you've updated all of these dependencies here? I'd presume that the old versions should work with 2018 edition?

We don't want to update dependencies in this PR though -- and lets actually whitelist src/test to be ignored by the tidy lint.

Reply:
No, i just updated the dependencies when i was trying to figure out why there's compilation error in 2018 - and it was just related to the removed `extern crate panic_halt;` mentioned above. Now I've rolled back the dependency changes.

I don't really have a good idea on how to whitelist the src/test directory... The existing code is using `Path::end_with` which doesn't work for intermediate path fragments...

Reply:
No, it's easy, i was just being silly. Done.